### PR TITLE
Running react-redux with Babel 6

### PR DESCRIPTION
**Pull request by martincik**
Refs #206

### CODE REVIEW

File: .babelrc [hunk 1]
```diff
@@ -1,4 +1,21 @@
 {
-  "stage": 0,
-  "loose": "all"
+  "presets": ["es2015", "stage-0", "react"],
+  "plugins": [
```
Comment (chentsulin) [inline .babelrc]: Why we need to add those plugins? They should already included in presets.

Comment (martincik) [inline .babelrc]: Refactored the code. Thanks. :+1:

File: test/components/connect.spec.js [hunk 2]
```diff
@@ -985,13 +985,14 @@ describe('React', () => {
 
       function imitateHotReloading(TargetClass, SourceClass) {
         // Crude imitation of hot reloading that does the job
-        Object.keys(SourceClass.prototype).filter(key =>
+        Object.getOwnPropertyNames(SourceClass.prototype).filter(key =>
           typeof SourceClass.prototype[key] === 'function'
         ).forEach(key => {
-          if (key !== 'render') {
+          if (key !== 'render' && key !== 'constructor') {
```
Comment (chentsulin) [inline test/components/connect.spec.js]: Is modifications in this file relative to babel6?

Comment (martincik) [inline test/components/connect.spec.js]: Yes.

**Comment by satazor** [comment]
+1

**Comment by ellbee** [comment]
@gaearon Do we want loose mode for IE8? React-Redux previously used it but this PR does not.

**Comment by gaearon** [comment]
How is loose mode related to IE8? I thought it was about improving perf.

**Comment by gaearon** [comment]
Either way I'd prefer adding loose mode back :-). I don't know how to do it though.

**Comment by ellbee** [comment]
When not in loose mode Babel uses Object.defineProperty. You are right about the perf too though.

There is a preset for loose mode. The Babel 6 PR in redux has it.

**Comment by gaearon** [comment]
Can you please fix it here?

**Comment by ellbee** [comment]
Yep, give me a minute.

**Comment by gaearon** [comment]
@ellbee Any other work there, or can I release 4.0.1 that doesn't have `.babelrc`?

**Comment by ellbee** [comment]
@gaearon Go for it.

**Comment by gaearon** [comment]
Out with 4.0.1 :+1:

**Pull request merged**